### ISSUE TITLE
JACK is missing in AppImages

### ISSUE THREAD

**Issue by morevnaproject** [synfig/synfig#395]
At the moment the JACK support is missing in AppImage builds.

See mkjack() function in ```autobild/fedora-crosscompile-linux.sh``` script for reference.

The following lines in launch.sh script of Appimage file is also relevant:
```
# Check if this system have JACK installed
if ( ! ldconfig -p | grep libjack.so >/dev/null ) || ( ! which jackd >/dev/null ) ; then
        # No JACK, so disable this functionality.
        # (The bundled libjack won't work correctly anyway).
        export SYNFIG_DISABLE_JACK=1
        export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:${BASE_DIR}/lib.extra"
fi
```

**Comment by blackwarthog** [comment]
done

**Comment by morevnaproject** [comment]
The fix is shipped with Synfig Studio 1.3.4 - https://morevnaproject.org/2017/08/10/synfig-studio-1-3-4-released/
